src/applications/disability-benefits/all-claims/pages/vaMedicalRecords.js

```diff
@@ -1,7 +1,7 @@
 import merge from 'lodash/merge';
 import fullSchema from '../config/schema';
 import { uiSchema as autoSuggestUiSchema } from 'us-forms-system/lib/js/definitions/autosuggest';
-import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+import dateRangeUI from 'us-forms-system/lib/js/definitions/monthYearRange';
```

Comment:
Huh. Didn't even know that was a thing.

Reply:
Me neither! I was fully prepared to make it a thing when I decided I had better check to see if we've done that already. Knocked the estimate down to a 1 since it is a thing already. :smile: